# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * account_yodlee
# 
# Translators:
# Bagavathikumar Ramakrishnan <bagavathikumar@gmail.com>, 2019
# Alagappan Karthikeyan <me@karthik.sg>, 2019
# 
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 12.0+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-01-15 15:18+0000\n"
"PO-Revision-Date: 2018-08-24 11:37+0000\n"
"Last-Translator: Alagappan Karthikeyan <me@karthik.sg>, 2019\n"
"Language-Team: Tamil (https://www.transifex.com/odoo/teams/41243/ta/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: ta\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:103
#: code:addons/account_yodlee/models/yodlee.py:110
#, python-format
msgid "(Diagnostic: %r for URL %s)"
msgstr "கண்டறிதல்: URL க்கான% r)"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:242
#, python-format
msgid "Account"
msgstr "கணக்கு"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:335
#, python-format
msgid ""
"Account Already Registered (436):Your Auto-Registration could not be "
"completed because the site reports that your account is already registered."
"  Please log in to the site to confirm and then complete the site addition "
"process with the correct login information."
msgstr ""
"ஏற்கனவே பதிவுசெய்யப்பட்ட கணக்கு (436): உங்கள் கணக்கு ஏற்கனவே "
"பதிவுசெய்யப்பட்டதாகத் தெரிவித்ததால் உங்கள் ஆட்டோ-பதிவு முடிக்கப்படவில்லை. "
"சரியான உள்நுழைவு தகவலுடன் தளத்தை கூடுதலாக செயல்படுத்துவதை உறுதிப்படுத்த "
"பின்னர் தளத்தில் உள்நுழைக."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:322
#, python-format
msgid ""
"Account Locked (407): We could not update your account because it appears "
"your account has been locked. This usually results from too many "
"unsuccessful login attempts in a short period of time. Please visit the site"
" or contact its customer support to resolve this issue.  Once done, please "
"update your account credentials in case they are changed."
msgstr ""
"கணக்கு பூட்டப்பட்டது (407): உங்கள் கணக்கு பூட்டப்பட்டிருப்பதால் உங்கள் "
"கணக்கை புதுப்பிக்க முடியவில்லை. இது ஒரு குறுகிய காலத்தில் பல தவறான உள்நுழைவு"
" முயற்சிகள் மூலம் வழக்கமாக விளைகிறது. இந்தச் சிக்கலைத் தீர்க்க தளத்தை "
"பார்வையிடுக அல்லது அதன் வாடிக்கையாளர் ஆதரவைத் தொடர்புகொள்ளவும். ஒருமுறை "
"முடித்துவிட்டால், உங்கள் கணக்கு சான்றுகளை அவர்கள் மாற்றியிருந்தால் "
"புதுப்பிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:349
#, python-format
msgid ""
"Account Not Found(408): We're sorry, we couldn't find any accounts for you "
"at the site. Please log in at the site and confirm that your account is set "
"up, then try again."
msgstr ""
"கணக்கைக் காணவில்லை (408): மன்னிக்கவும், தளத்தில் நீங்கள் எந்த கணக்குகளையும் "
"கண்டுபிடிக்க முடியவில்லை. தளத்தில் உள்நுழைந்து, உங்கள் கணக்கு "
"அமைக்கப்பட்டுள்ளதை உறுதிப்படுத்தவும், மீண்டும் முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:327
#, python-format
msgid ""
"Account Reported Closed (422): We were unable to update your account "
"information because it appears one or more of your related accounts have "
"been closed.  Please deactivate or delete the relevant account and try "
"again."
msgstr ""
"கணக்கு அறிக்கை மூடப்பட்டது (422): உங்கள் கணக்குத் தகவலை புதுப்பிக்க "
"முடியவில்லை, ஏனெனில் உங்கள் ஒன்று அல்லது அதற்கு மேற்பட்ட தொடர்புடைய "
"கணக்குகள் மூடப்பட்டுள்ளன. தொடர்புடைய கணக்குகளை செயலிழக்க அல்லது நீக்கவும், "
"மீண்டும் முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:324
#, python-format
msgid ""
"Account Type Not Supported(417):The type of account we found is not "
"currently supported.  Please remove this site and add as a  manual account."
msgstr ""
"கணக்கு வகை ஆதரவு இல்லை (417): நாங்கள் கண்டறிந்த கணக்கின் வகை தற்போது "
"ஆதரிக்கப்படவில்லை. தயவு செய்து இந்த தளத்தை நீக்கவும் மற்றும் கையேடு கணக்காக "
"சேர்க்கவும்."

#. module: account_yodlee
#: model:ir.model.fields,help:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Active/Inactive on Yodlee system"
msgstr "செயலில் / செயலற்ற செயல் முறைமை"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:357
#, python-format
msgid "An Error has occurred (code %s)"
msgstr "ஒரு பிழை ஏற்பட்டது (குறியீடு)"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_res_company
msgid "Companies"
msgstr "நிறுவனங்கள்"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:199
#, python-format
msgid "Could not make sense of parameters: %s"
msgstr "அளவுருக்கள் உணர முடியவில்லை:"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:339
#, python-format
msgid ""
"Credential Re-Verification Required (402): We could not update your account "
"because your username and/or password were reported to be incorrect.  Please"
" re-verify your username and password."
msgstr ""
"தகுதி மறு சரிபார்ப்பு தேவை (402): உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை "
"ஏனெனில் உங்கள் பயனர்பெயர் மற்றும் / அல்லது கடவுச்சொல் தவறானது என்று "
"புகாரளிக்கப்பட்டது. உங்கள் பயனர்பெயர் மற்றும் கடவுச்சொல்லை மீண்டும் "
"சரிபார்க்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:345
#, python-format
msgid ""
"Credential Re-Verification Required (420):The site has merged with another. "
"Please re-verify your credentials at the site and update the same."
msgstr ""
"தகுதி மறு சரிபார்ப்பு தேவை (420): தளம் மற்றொரு இணைந்திருக்கிறது. தளத்தில் "
"உங்கள் சான்றுகளை மீண்டும் சரிபார்க்கவும், அதே போல் புதுப்பிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:228
#, python-format
msgid "Error %s, message: %s"
msgstr "பிழை செய்தி:"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:102
#, python-format
msgid "Error %s, message: %s, reference code: %s"
msgstr "பிழை, செய்தி:, குறிப்பு குறியீடு"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:318
#, python-format
msgid ""
"Home Value Not Found (511): The site is unable to provide home value for "
"your property. We suggest you to delete this site."
msgstr ""
"முகப்பு மதிப்பு காணப்படவில்லை (511): தளத்தில் உங்கள் சொத்துக்கான வீடான "
"மதிப்பை வழங்க முடியவில்லை. இந்த தளத்தை நீக்குமாறு நாங்கள் பரிந்துரைக்கிறோம்."

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_journal
msgid "Interface for Online Account Journal"
msgstr "ஆன்லைன் கணக்கு ஜர்னலுக்கான இடைமுகம்"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:326
#, python-format
msgid ""
"Invalid Language Setting (421): The language setting for your site account "
"is not English. Please visit the site and change the language setting to "
"English."
msgstr ""
"தவறான மொழி அமைவு (421): உங்கள் தள கணக்குக்கான மொழி அமைப்பு ஆங்கிலம் அல்ல. "
"தயவுசெய்து தளத்தைப் பார்வையிடவும், ஆங்கில அமைப்பை ஆங்கில மொழியில் மாற்றவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:341
#, python-format
msgid ""
"MFA error: Additional Authentication Enrollment Required (521) : Please "
"enroll in the new security authentication system, <Account Name> has "
"introduced. Ensure your account settings in <Cobrand> are updated with this "
"information."
msgstr ""
"MFA பிழை: கூடுதல் அங்கீகார பதிவு தேவை (521): தயவுசெய்து புதிய பாதுகாப்பு "
"அங்கீகார அமைப்பில் சேரவும். உங்கள் கணக்கு அமைப்புகளில் இந்த தகவலுடன் "
"புதுப்பிக்கப்பட்டுள்ளன என்பதை உறுதிசெய்யவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:344
#, python-format
msgid ""
"MFA error: Authentication Information Expired (524):We're sorry, the "
"authentication information you provided has expired. Please try again."
msgstr ""
"MFA பிழை: அங்கீகார தகவல் காலாவதியானது (524): மன்னிக்கவும், நீங்கள் வழங்கிய "
"அங்கீகார தகவல் காலாவதியானது. தயவுசெய்து மீண்டும் முயற்சி செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:340
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (520):We're sorry, the site "
"indicates that the additional authentication information you provided is "
"incorrect. Please try updating your account again."
msgstr ""
"MFA பிழை: அங்கீகாரத் தகவல் தவறானது (520): மன்னிக்கவும், நீங்கள் வழங்கிய "
"கூடுதல் அங்கீகாரத் தகவல் தவறானது என்று தளம் குறிப்பிடுகிறது. உங்கள் கணக்கை "
"மீண்டும் புதுப்பித்துப் பார்க்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:343
#, python-format
msgid ""
"MFA error: Authentication Information Incorrect (523):We're sorry, the "
"authentication information you  provided is incorrect. Please try again."
msgstr ""
"MFA பிழை: அங்கீகாரத் தகவல் தவறானது (523): மன்னிக்கவும், நீங்கள் வழங்கிய "
"அங்கீகாரத் தகவல் தவறானது. தயவுசெய்து மீண்டும் முயற்சி செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:339
#, python-format
msgid ""
"MFA error: Authentication Information Required (519): Your account was not "
"updated as your authentication information like security question and answer"
" was unavailable or incomplete. Please update your account settings."
msgstr ""
"MFA பிழை: அங்கீகாரத் தகவல் தவறானது (523): மன்னிக்கவும், நீங்கள் வழங்கிய "
"அங்கீகாரத் தகவல் தவறானது. தயவுசெய்து மீண்டும் முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:338
#, python-format
msgid ""
"MFA error: Authentication Information Unavailable (518):Your account was not"
" updated as the required additional authentication information was "
"unavailable. Please try now."
msgstr ""
"MFA பிழை: அங்கீகார தகவல் கிடைக்கவில்லை (518): தேவையான கூடுதல் அங்கீகாரத் "
"தகவல்கள் கிடைக்காததால் உங்கள் கணக்கு புதுப்பிக்கப்படவில்லை. தயவுசெய்து "
"இப்போது முயற்சி செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:345
#, python-format
msgid ""
"MFA error: Credential Re-Verification Required (526): We could not update "
"your account because your username/password or additional security "
"credentials are incorrect. Please try again."
msgstr ""
"MFA பிழை: தகுதி மறு சரிபார்ப்பு தேவை (526): உங்கள் கணக்கை புதுப்பிக்க "
"முடியவில்லை ஏனெனில் உங்கள் பயனர் பெயர் / கடவுச்சொல் அல்லது கூடுதல் "
"பாதுகாப்பு சான்றுகள் தவறானவை. தயவுசெய்து மீண்டும் முயற்சி செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:356
#, python-format
msgid ""
"MFA error: Problem Updating Account (525): We could not update your account "
"for technical reasons. This type of error is usually resolved in a few days."
" We apologize for the inconvenience. Please try again later."
msgstr ""
"MFA பிழை: பிரச்சனை புதுப்பித்தல் கணக்கு (525): தொழில்நுட்ப காரணங்களுக்காக "
"உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை. இந்த வகை பிழை பொதுவாக ஒரு சில "
"நாட்களில் தீர்க்கப்படும். சிரமத்திற்கு வருந்துகிறோம். தயவுசெய்து பின்னர் "
"முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:342
#, python-format
msgid ""
"MFA error: Request Timed Out (522) :Your request has timed out as the "
"required security information was unavailable or was not provided within the"
" expected time. Please try again."
msgstr ""
"MFA பிழை: கோரிக்கை காலாவதியானது (522): தேவையான பாதுகாப்பு தகவல் "
"கிடைக்கவில்லை அல்லது எதிர்பார்த்த நேரத்திற்குள் வழங்கப்படவில்லை, உங்கள் "
"கோரிக்கை நேரம் முடிந்தது. தயவுசெய்து மீண்டும் முயற்சி செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:354
#, python-format
msgid ""
"MFA error: Site Device Information Expired(509): We're sorry, we can't "
"update your account because your token is no longer valid at the site. "
"Please update your information and try again, or contact customer support."
msgstr ""
"MFA பிழை: தள சாதன தகவல் காலாவதியானது (509): மன்னிக்கவும், உங்கள் கணக்கை "
"புதுப்பிக்க முடியாது, ஏனெனில் உங்கள் டோக்கன் தளத்தில் இனி செல்லுபடியாகாது. "
"உங்கள் தகவலைப் புதுப்பித்து மீண்டும் முயற்சிக்கவும் அல்லது வாடிக்கையாளர் "
"ஆதரவைத் தொடர்பு கொள்ளவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:310
#, python-format
msgid ""
"Multiple User Logins(416): We attempted to update your account, but another "
"session was already established at the same time.  If you are currently "
"logged on to this account directly, please log off and try after some time"
msgstr ""
"பல பயனர் உள்நுழைவுகள் (416): உங்கள் கணக்கை புதுப்பிக்க முயற்சித்தோம், ஆனால் "
"மற்றொரு அமர்வு ஏற்கனவே அதே நேரத்தில் நிறுவப்பட்டது. நீங்கள் தற்போது இந்த "
"கணக்கில் நேரடியாக உள்நுழைந்திருந்தால், தயவு செய்து புகுபதிகை செய்து சிறிது "
"நேரம் கழித்து முயற்சிக்கவும்"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:336
#, python-format
msgid ""
"New Login Information Required(506):We're sorry, to log in to this site, you"
" need to provide additional information. Please update your account and try "
"again."
msgstr ""
"புதிய தேதி தகவல் தேவை (506): மன்னிக்கவும், இந்த தளத்திற்கு உள்நுழைய, கூடுதல்"
" தகவல் வழங்க வேண்டும். உங்கள் கணக்கை புதுப்பித்து மீண்டும் முயற்சி செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:312
#, python-format
msgid ""
"No Account Found (423): We were unable to detect an account. Please verify "
"that you account information is available at this time and If the problem "
"persists, please contact customer support at online@odoo.com for further "
"assistance."
msgstr ""
"கணக்கு எதுவும் இல்லை (423): ஒரு கணக்கை நாங்கள் கண்டுபிடிக்க முடியவில்லை. "
"கணக்கு விவரங்கள் இந்த நேரத்தில் கிடைக்கின்றனவா என்பதை சரிபாருங்கள் மற்றும் "
"சிக்கல் தொடர்ந்தால், தயவுசெய்து வாடிக்கையாளர் ஆதரவை ஆன்லைனில் "
"சமர்ப்பிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:337
#, python-format
msgid ""
"No Payees Found(512):Your request cannot be completed as no payees were "
"found in your account."
msgstr ""
"எந்த ஊதியமும் இல்லை (512): உங்கள் கணக்கில் பணம் எதுவும் கிடைக்காததால் உங்கள்"
" கோரிக்கையை முடிக்க முடியாது."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:194
#, python-format
msgid "Online institution"
msgstr "ஆன்லைன் நிறுவனம்"

#. module: account_yodlee
#: selection:account.online.provider,provider_type:0
msgid "Plaid"
msgstr "பிளேசிட்"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:48
#, python-format
msgid "Please configure an email in the company settings."
msgstr "நிறுவன அமைப்புகளில் ஒரு மின்னஞ்சலை உள்ளிடுக."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:321
#, python-format
msgid ""
"Problem Updating Account (406): We could not update your account because the"
" site requires you to perform some additional action. Please visit the site "
"or contact its customer support to resolve this issue. Once done, please "
"update your account credentials in case they are changed else try again."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (406): உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை, "
"ஏனெனில் சில கூடுதல் நடவடிக்கைகளை நீங்கள் செய்ய வேண்டும். இந்தச் சிக்கலைத் "
"தீர்க்க தளத்தை பார்வையிடுக அல்லது அதன் வாடிக்கையாளர் ஆதரவைத் "
"தொடர்புகொள்ளவும். ஒருமுறை முடித்துவிட்டால், உங்கள் கணக்கின் சான்றுகளை "
"அவர்கள் மீண்டும் மாற்றினால், மீண்டும் முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:355
#, python-format
msgid ""
"Problem Updating Account (517): We are sorry, there was a technical problem "
"updating your account. Please try again."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (517): மன்னிக்கவும், உங்கள் கணக்கை "
"மேம்படுத்தும் தொழில்நுட்ப சிக்கல் ஏற்பட்டது. தயவுசெய்து மீண்டும் முயற்சி "
"செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:346
#, python-format
msgid ""
"Problem Updating Account(401):We're sorry, your request timed out. Please "
"try again."
msgstr ""
"கணக்கு புதுப்பித்தல் கணக்கு (401): மன்னிக்கவும், உங்கள் கோரிக்கை நேரம் "
"முடிந்தது. தயவுசெய்து மீண்டும் முயற்சி செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:347
#, python-format
msgid ""
"Problem Updating Account(403):We're sorry, there was a technical problem "
"updating your account. This kind of error is usually resolved in a few days."
" Please try again later."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (403): மன்னிக்கவும், உங்கள் கணக்கு "
"புதுப்பிப்பதற்கான ஒரு தொழில்நுட்ப சிக்கல் ஏற்பட்டது. இந்த வகையான பிழை "
"பொதுவாக ஒரு சில நாட்களில் தீர்க்கப்படும். தயவுசெய்து பின்னர் முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:348
#, python-format
msgid ""
"Problem Updating Account(404):We're sorry, there was a technical problem "
"updating your account. Please try again later."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (404): மன்னிக்கவும், உங்கள் கணக்கை "
"மேம்படுத்தும் ஒரு தொழில்நுட்ப சிக்கல் ஏற்பட்டது. தயவுசெய்து பின்னர் "
"முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:306
#, python-format
msgid ""
"Problem Updating Account(409): We could not update your account because the "
"end site is experiencing technical difficulties."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (409): இறுதித் தளம் தொழில்நுட்ப சிக்கல்களை "
"எதிர்கொண்டதால் உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:308
#, python-format
msgid ""
"Problem Updating Account(412): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"பிரச்சனையை புதுப்பித்தல் கணக்கு (412): தளம் தொழில்நுட்ப சிக்கல்களை "
"எதிர்கொண்டதால் உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:350
#, python-format
msgid ""
"Problem Updating Account(413):We're sorry, we couldn't update your account "
"at the site because of a technical issue. This type of problem is usually "
"resolved in a few days. Please try again later."
msgstr ""
"சிக்கலைப் புதுப்பித்தல் கணக்கு (413): வருந்துகிறோம், தொழில்நுட்ப சிக்கல் "
"காரணமாக உங்கள் கணக்கை தளத்தில் புதுப்பிக்க முடியவில்லை. இந்த வகை பிரச்சனை "
"பொதுவாக சில நாட்களில் தீர்க்கப்படும். தயவுசெய்து பின்னர் முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:309
#, python-format
msgid ""
"Problem Updating Account(415): We could not update your account because the "
"site is experiencing technical difficulties."
msgstr ""
"பிரச்சனை புதுப்பிப்பு கணக்கு (415): உங்கள் தொழில்நுட்பத்தை சிக்கலாக்குவதால்,"
" உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:311
#, python-format
msgid ""
"Problem Updating Account(418): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (418): தளம் உங்கள் தொழில்நுட்பத்தைச் "
"சிக்கலாக்கியதால், உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை. தயவுசெய்து பின்னர் "
"முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:351
#, python-format
msgid ""
"Problem Updating Account(419):We're sorry, we couldn't update your account "
"because of unexpected variations at the site. This kind of problem is "
"usually resolved in a few days. Please try again later."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (419): மன்னிக்கவும், தளத்தின் எதிர்பாராத "
"மாறுபாடுகளின் காரணமாக உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை. இந்த வகையான "
"பிரச்சனை பொதுவாக சில நாட்களில் தீர்க்கப்படும். தயவுசெய்து பின்னர் "
"முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:314
#, python-format
msgid ""
"Problem Updating Account(425): We could not update your account because the "
"site is experiencing technical difficulties. Please try later."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (425): தளம் தொழில்நுட்ப சிக்கல்களை "
"எதிர்கொண்டதால் உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை. தயவுசெய்து பின்னர் "
"முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:315
#, python-format
msgid ""
"Problem Updating Account(426): We could not update your account for "
"technical reasons. This type of error is usually resolved in a few days. We "
"apologize for the inconvenience."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (426): தொழில்நுட்ப காரணங்களுக்காக உங்கள் "
"கணக்கை புதுப்பிக்க முடியவில்லை. இந்த வகை பிழை பொதுவாக ஒரு சில நாட்களில் "
"தீர்க்கப்படும். சிரமத்திற்கு வருந்துகிறோம்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:352
#, python-format
msgid ""
"Problem Updating Account(507):We're sorry, Yodlee has just started providing"
" data updates for this site, and it may take a few days to be successful as "
"we get started. Please try again later."
msgstr ""
"பிரச்சனை புதுப்பித்தல் கணக்கு (507): மன்னிக்கவும், Yodlee இந்த தளத்திற்கான "
"தரவு புதுப்பிப்புகளை வழங்கத் தொடங்கியுள்ளது, மேலும் நாங்கள் தொடங்குவதற்குள் "
"சில நாட்கள் வெற்றிகரமாக ஆகலாம். தயவுசெய்து பின்னர் முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:317
#, python-format
msgid ""
"Property Record Not Found (510): The site is unable to find any property "
"information for your address. Please verify if the property address you have"
" provided is correct."
msgstr ""
"சொத்து பதிவு இல்லை (510): உங்கள் முகவரிக்கு எந்த சொத்து தகவலையும் "
"கண்டுபிடிக்க முடியவில்லை. நீங்கள் வழங்கிய சொத்து முகவரி சரியானதா என "
"சரிபார்க்கவும்."

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_provider__provider_type
msgid "Provider Type"
msgstr "வழங்குபவர் வகை"

#. module: account_yodlee
#: model:ir.model,name:account_yodlee.model_account_online_provider
msgid "Provider for online account synchronization"
msgstr "ஆன்லைன் கணக்கு ஒத்திசைவு வழங்குபவர்"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:330
#, python-format
msgid ""
"Re-Verification Required (429): We could not update your account due to the "
"site requiring you to verify your personal information. Please log in to the"
" site and update the fields required."
msgstr ""
"மீண்டும் சரிபார்ப்பு தேவை (429): உங்கள் தனிப்பட்ட தகவலைச் சரிபார்க்க நீங்கள்"
" தேவைப்படும் தளத்தின் காரணமாக உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை. "
"தயவுசெய்து தளத்திற்கு உள்நுழைந்து தேவையான புலங்களை புதுப்பிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:328
#, python-format
msgid ""
"Re-verification Required (427): We could not update your account due to the "
"site requiring you to view a new promotion. Please log in to the site and "
"click through to your account overview page to update the account.  We "
"apologize for the inconvenience."
msgstr ""
"மறு சரிபார்ப்பு தேவை (427): நீங்கள் புதிய விளம்பரத்தை பார்வையிட வேண்டிய தளம்"
" காரணமாக உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை. தயவுசெய்து தளத்தில் "
"உள்நுழைந்து, கணக்கைப் புதுப்பிக்க உங்கள் கணக்கின் கண்ணோட்டம் பக்கத்திற்குச் "
"செல்லவும். சிரமத்திற்கு வருந்துகிறோம்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:329
#, python-format
msgid ""
"Re-verification Required (428): We could not update your account due to the "
"site requiring you to accept a new Terms & Conditions. Please log in to the "
"site and read and accept the T&C."
msgstr ""
"மறு சரிபார்ப்பு தேவை (428): புதிய விதிமுறைகளையும் நிபந்தனைகளையும் நீங்கள் "
"ஏற்க வேண்டிய தளம் காரணமாக உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை. தயவு செய்து"
" தளத்திற்குச் சென்று T & C ஐ படித்து ஏற்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:332
#, python-format
msgid ""
"Registration Requires Attention (433): Auto registration is not complete. "
"Please complete your registration at the end site. Once completed, please "
"complete adding this account."
msgstr ""
"பதிவு கவனத்திற்கு (433) தேவைப்படுகிறது: ஆட்டோ பதிவு முழுமை பெறவில்லை. தயவு "
"செய்து உங்கள் பதிவை இறுதி தளத்தில் முடிக்க வேண்டும். முடிந்ததும், இந்த "
"கணக்கைச் சேர்த்து முடிக்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:333
#, python-format
msgid ""
"Registration Requires Attention (434): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"பதிவு கவனத்திற்கு (434) தேவைப்படுகிறது: உங்கள் ஆட்டோ-பதிவு முடிக்கப்படாது "
"மேலும் உங்களிடமிருந்து மேலும் உள்ளீடு தேவைப்படுகிறது. செயல்முறை முடிக்க "
"உங்கள் பதிவு தகவலை மீண்டும் சரிபார்க்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:334
#, python-format
msgid ""
"Registration Requires Attention (435): Your Auto-Registration could not be "
"completed and requires further input from you.  Please re-verify your "
"registration information to complete the process."
msgstr ""
"பதிவு செய்ய வேண்டிய அவசியம் (435): உங்கள் ஆட்டோ-பதிவு முடிக்கப்படாது, "
"உங்களிடமிருந்து மேலும் உள்ளீடு தேவைப்படுகிறது. செயல்முறை முடிக்க உங்கள் "
"பதிவு தகவலை மீண்டும் சரிபார்க்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:353
#, python-format
msgid ""
"Request Timed Out (508): We are sorry, your request timed out due to "
"technical reasons. Please try again."
msgstr ""
"கோரிக்கை காலாவதியானது (508): நாங்கள் வருந்துகிறோம், தொழில்நுட்ப காரணங்களால் "
"உங்கள் கோரிக்கை முடிந்துவிட்டது. தயவுசெய்து மீண்டும் முயற்சி செய்க."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:323
#, python-format
msgid ""
"Requested Account Type Not Found (414): We could not find your requested "
"account. You may have selected a similar site under a different category by "
"accident in which case you should select the correct site."
msgstr ""
"கோரப்பட்ட கணக்கு வகை காணப்படவில்லை (414): உங்கள் கோரப்பட்ட கணக்கை நாங்கள் "
"கண்டுபிடிக்க முடியவில்லை. விபத்து மூலம் வெவ்வேறு வகையின் கீழ் நீங்கள் இதே "
"போன்ற தளத்தைத் தேர்ந்தெடுத்திருக்கலாம், அதில் நீங்கள் சரியான தளத்தைத் "
"தேர்ந்தெடுக்க வேண்டும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:313
#, python-format
msgid ""
"Site Down for Maintenance(424):We were unable to update your account as the "
"site is temporarily down for maintenance. We apologize for the "
"inconvenience.  This problem is typically resolved in a few hours. Please "
"try later."
msgstr ""
"தளம் கீழே பராமரிப்புக்காக (424): பராமரிப்பிற்காக தளம் தற்காலிகமாக கீழே "
"இறங்கும்போது உங்கள் கணக்கை புதுப்பிக்க முடியவில்லை. சிரமத்திற்கு "
"வருந்துகிறோம். இந்த பிரச்சனை சில மணிநேரங்களில் தீர்மானிக்கப்படுகிறது. "
"தயவுசெய்து பின்னர் முயற்சிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:307
#, python-format
msgid ""
"Site No Longer Available (411):The site no longer provides online services "
"to its customers.  Please delete this account."
msgstr ""
"தளம் இல்லை இனி கிடைக்கும் (411): தளத்தில் இனி அதன் வாடிக்கையாளர்களுக்கு "
"ஆன்லைன் சேவைகளை வழங்குகிறது. தயவுசெய்து இந்த கணக்கை அழிக்கவும்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:331
#, python-format
msgid ""
"Site No Longer Supported (430):This site is no longer supported for data "
"updates. Please deactivate or delete your account. We apologize for the "
"inconvenience."
msgstr ""
"தளம் இல்லை நீண்ட ஆதரவு (430): இந்தத் தளம் தரவு புதுப்பிப்புகளுக்கு "
"ஆதரிக்கப்படாது. உங்கள் கணக்கை செயலிழக்க அல்லது நீக்கவும். சிரமத்திற்கு "
"வருந்துகிறோம்."

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:316
#, python-format
msgid ""
"Site Not Supported (505): We currently does not support the security system "
"used by this site. We apologize for any inconvenience. Check back "
"periodically if this situation has changed."
msgstr ""
"தள ஆதரவு இல்லை (505): நாங்கள் தற்போது இந்த தளத்தின் மூலம் பயன்படுத்தும் "
"பாதுகாப்பு முறையை ஆதரிக்கவில்லை. சிரமத்திற்கு வருந்துகிறோம். இந்த நிலைமை "
"மாறியிருந்தால் அவ்வப்போது சரிபார்க்கவும்."

#. module: account_yodlee
#: model:ir.actions.client,name:account_yodlee.yodlee_widget
msgid "Synchronization process completed"
msgstr "ஒத்திசைத்தல் செயல்முறை முடிந்தது"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:60
#: code:addons/account_yodlee/models/yodlee.py:73
#: code:addons/account_yodlee/models/yodlee.py:87
#: code:addons/account_yodlee/models/yodlee.py:135
#, python-format
msgid "Timeout: the server did not reply within 30s"
msgstr "காலாவதியானது: 30 களில் சர்வர் பதிலளிக்கவில்லை"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:320
#, python-format
msgid ""
"Update Request Canceled(405):Your account was not updated because you "
"canceled the request."
msgstr ""
"புதுப்பிக்கப்பட்ட கோரிக்கை ரத்து செய்யப்பட்டது (405): நீங்கள் கோரிக்கையை "
"ரத்து செய்ததால் உங்கள் கணக்கு புதுப்பிக்கப்படவில்லை."

#. module: account_yodlee
#: selection:account.online.provider,provider_type:0
msgid "Yodlee"
msgstr ""

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_account_status
msgid "Yodlee Account Status"
msgstr "Yodlee கணக்கு நிலை"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_account_online_journal__yodlee_status_code
msgid "Yodlee Status Code"
msgstr "Yodlee நிலைமை கோட்"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_access_token
msgid "Yodlee access token"
msgstr "Yodlee அணுகல் டோக்கன்"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_login
msgid "Yodlee login"
msgstr ""

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_user_password
msgid "Yodlee password"
msgstr "Yodlee கடவுச்சொல்"

#. module: account_yodlee
#: model:ir.model.fields,field_description:account_yodlee.field_res_company__yodlee_access_token
msgid "access_token"
msgstr "ACCESS_TOKEN"

#. module: account_yodlee
#: code:addons/account_yodlee/models/yodlee.py:297
#, python-format
msgid "message: %s"
msgstr "செய்தி:"
